locales/sv_SE/pcsx2_Iconized.po

```diff
@@ -425,18 +442,19 @@ msgid ""
 msgstr ""
 "Markera för att få Muspekaren att döljas då den är i ett GS-fönster;\n"
 " användbart vid brukande av musen som främsta styrmojäng för spelande.\n"
-"Som praxis gömmer sig Muspekaren automatiskt efter 2 sekunders inaktivitet."
+"Som standard gömmer sig Muspekaren automatiskt\n"
+" efter 2 sekunders inaktivitet."
 
 #: pcsx2/gui/Panels/GSWindowPanel.cpp:84
 msgid ""
 "Enables automatic mode switch to fullscreen when starting or resuming "
 "emulation. You can still toggle fullscreen display at any time using alt-"
 "enter."
 msgstr ""
-"Möjliggör automatiskt lägesbyte till Helskärm då emulering påbörjas eller "
-"återupptas.\n"
-"Ni kan fortfarande växla mellan Helskärm och Inramningsläge genom att trycka "
-"''Alt'' + ''Enter''."
+"Möjliggör automatiskt lägesbyte till Helskärm då emulering\n"
```

Comment:
```suggestion
"Möjliggör automatiskt lägesbyte till helskärmsläge då emulering\n"
```

Reply:
To have "läge" twice is not called for.
The sentence is quite clear, I think.